dietpi/dietpi-cleaner

```diff
@@ -87,7 +87,7 @@
 
 					G_WHIP_MSG "$G_PROGRAM_NAME is a program that allows you to remove unwanted junk from your DietPi system, freeing up filesystem space in the process.\n
 Simply enable the cleaners you require, then select \"Test\" to see what will happen, without modifying your system.\nOnce your satisfied with the Test results, select \"Run\" to clean your system.\n
-Further information:\n - https://dietpi.com/phpbb/viewtopic.php?p=623#p623"
+Further information:\n - http://localhost:8000/dietpi_tools/#dietpi-cleaner"
```

Comment:
You might want to replace this with `https://dietpi.com/docs/dietpi_tools/#dietpi-cleaner`.